cmd/gpu_plugin/gpu_plugin.go

```diff
@@ -627,7 +767,10 @@ func main() {
 	flag.StringVar(&prefix, "prefix", "", "Prefix for devfs & sysfs paths")
 	flag.BoolVar(&opts.enableMonitoring, "enable-monitoring", false, "whether to enable '*_monitoring' (= all GPUs) resource")
 	flag.BoolVar(&opts.resourceManagement, "resource-manager", false, "fractional GPU resource management")
+	flag.BoolVar(&opts.levelzero, "levelzero", false, "Enable retrieval of data from Level-ero interface")
```

Comment:
```suggestion
	flag.BoolVar(&opts.levelzero, "levelzero", false, "Enable retrieval of data from Level-Zero interface")
```

But I think rather than having this option, L0 should be enabled automatically if features needing it are enabled (WSL, temp limit, or GPU error checks).   And if L0 is not available when those features are requested, plugin should either fail, or log a warning.

Reply:
That is a good point. I'll rewrite this so that each feature has a enable flag.